src/main/java/io/github/ascopes/protobufmavenplugin/AbstractGenerateMojo.java

```diff
@@ -432,7 +431,19 @@ public void execute() throws MojoExecutionException, MojoFailureException {
     }
   }
 
-  /**
+  private Path outputDirectory() {
+    return Optional.ofNullable(outputDirectory).map(File::toPath).orElseGet(() -> defaultOutputDirectory(session));
+  }
+
+  private Collection<Path> sourceDirectories() {
+      if(sourceDirectories != null){
```

Comment:
would it be possible to run `./mvnw verify` locally just to double check checkstyle succeeds?